ZAPD/ZCollision.cpp

```diff
@@ -106,7 +123,7 @@ void ZCollisionHeader::DeclareReferences(const std::string& prefix)
 
 		parent->AddDeclarationArray(
 			waterBoxSegmentOffset, DeclarationAlignment::Align4, 16 * waterBoxes.size(), "WaterBox",
-			StringHelper::Sprintf("%s_waterBoxes_%06X", auxName.c_str(), waterBoxSegmentOffset),
+			StringHelper::Sprintf("%sWaterBoxes", auxName.c_str()),
```

Comment:
I'm not sure it's an advantage to drop the offset: might be helpful for debugging?